src/GrpcProtos/capture.proto

```diff
@@ -273,40 +273,50 @@ message ModuleUpdateEvent {
 
 message ClientCaptureEvent {
   oneof event {
+    // Note that field numbers from 1-15 take only 1 byte to encode
+    // https://developers.google.com/protocol-buffers/docs/proto#assigning_field_numbers
+    // use them for high frequency events. For the rest please assign
+    // numbers starting with 16.
+    //
     // Please keep these alphabetically ordered.
-    AddressInfo address_info = 1;
-    CallstackSample callstack_sample = 2;
-    FunctionCall function_call = 3;
-    GpuJob gpu_job = 4;
-    GpuQueueSubmission gpu_queue_submission = 5;
-    InternedCallstack interned_callstack = 6;
-    InternedString interned_string = 7;
-    InternedTracepointInfo interned_tracepoint_info = 8;
-    IntrospectionScope introspection_scope = 9;
-    ModuleUpdateEvent module_update_event = 10;
-    SchedulingSlice scheduling_slice = 11;
-    ThreadName thread_name = 12;
-    ThreadStateSlice thread_state_slice = 13;
-    TracepointEvent tracepoint_event = 14;
+    AddressInfo address_info = 16;
+    CallstackSample callstack_sample = 1;
+    FunctionCall function_call = 2;
+    GpuJob gpu_job = 3;
+    GpuQueueSubmission gpu_queue_submission = 4;
+    InternedCallstack interned_callstack = 17;
```

Comment:
I am not quite sure about this event.

Reply:
Me neither... One would expect low frequency because it's interned, but of course it's easy to have many different callstacks. Does this become more or less critical once we increase the sampling frequency with frame pointers?

Reply:
Still unsure but moved to 1-15 range.